pkgs/development/libraries/libspatialaudio/default.nix

```diff
@@ -0,0 +1,21 @@
+{ lib, stdenv, fetchFromGitHub, cmake, libmysofa, zlib }:
+stdenv.mkDerivation {
+  name = "libspatialaudio";
+  src = fetchFromGitHub {
+    owner = "videolabs";
+    repo = "libspatialaudio";
+    rev = "0.3.0";
+    hash = "sha256-sPnQPD41AceXM4uGqWXMYhuQv0TUkA6TZP8ChxUFIoI=";
+  };
+  buildInputs = [ cmake libmysofa zlib ];
```

Comment:
Build tools like `cmake` should go to `nativeBuildInputs`, see https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/build-tools-in-build-inputs.md.

Reply:
Fixed, thanks!